frontEnd/src/components/Header.vue

```diff
@@ -166,7 +172,7 @@ import * as cookies from 'vue-cookies'
         },
         beforeDestroy() {
           //destroy cookie
-          cookies.remove("AuthCookie");
+          cookies.remove(this.user.id);
```

Comment:
shouldnt this logic also go under the logout button as well?

Reply:
I don't think so since the logout always destroys anyways (in the same way that the clear interval is not necessary to be in the logout)